logging/src/logging.cc

```diff
@@ -85,7 +86,9 @@ namespace leatherman { namespace logging {
         // Note that this creates a locale that's not usable for testing, as it
         // only includes paths for install locations. This is intentional, to avoid leaving
         // searching paths that have unknown permissions.
-        dst.imbue(lth_locale::get_locale(locale, domain, {}));
+        if (use_locale) {
+            dst.imbue(lth_locale::get_locale(locale, domain, {}));
```

Comment:
Imbue here has no real impact. The only reason this line still exists is to call get_locale early, so we can setup the locale for the current project. We're already down this road, but it might've been better to just pull it out as a separate init step for projects.